main.c

```diff
@@ -459,6 +457,11 @@ void mainCallback(GPTDriver *_driver) {
   if (ledEnabled) {
 
     palClearLine(ledColumns[currentCol]);
+    for (int i = 0; i < NUM_ROW * 4; i++) {
+      if (i % 4 != 3) {
+        palClearLine(ledRows[i]);
```

Comment:
What we do here is not really important. The important thing is delay introduced by this instructions.

Reply:
LOL. Maybe then just stuff "nop"s there?

Reply:
It is hard to get the timing right. This accomplishes what is needed, and was simpler to understand :)